gradoop-flink/src/main/java/org/gradoop/flink/datagen/transactions/foodbroker/config/FoodBrokerConfig.java

```diff
@@ -87,14 +89,16 @@ public static FoodBrokerConfig fromJSONString(String configString)
    * @return list of String
    */
   public List<String> getStringValuesFromFile(String fileName) {
-    List<String> values = null;
+    String value = "";
+
+    InputStream inputStream = this.getClass().getResourceAsStream("/foodbroker/" + fileName);
     try {
-      values = FileUtils.readLines(FileUtils.getFile(
-        FoodBrokerConfig.class.getResource("/foodbroker/" + fileName).getFile()));
+      value = org.apache.commons.io.IOUtils.toString(inputStream, "UTF-8");
```

Comment:
If the file does not exist, the list could be returned empty causing the names (for example) to be empty and then the employees will not get any names assigned which is not wanted.
Also the resources have to be changed/removed to cause the npe.
So I do not think that handling the exception would be very beneficul.